fabtests/component/dmabuf-rdma/rocr.h

```diff
@@ -0,0 +1,92 @@
+
+#ifndef _DMABUF_RDMA_TESTS_XE_H_
+#define _DMABUF_RDMA_TESTS_XE_H_
+
+#include <stdint.h>
+#include "hmem.h"
+
+#include <hsa/hsa.h>
+#include <hsa/hsa_ext_amd.h>
+
+#define MAX_GPUS	(8)
+
+/*
+ * Buffer location and method of allocation
+ */
+enum {
+	MALLOC,	/* Host memory allocated via malloc and alike */
+	HOST,	/* Host memory allocated via zeMemAllocHost */
+	DEVICE,	/* Device memory allocated via zeMemAllocDevice */
+	SHARED	/* Shared memory allocated via zeMemAllocShared */
+};
+
+/*
+ * All information related to a buffer allocated via oneAPI L0 API.
+ */
+struct rocr_buf {
+	void			*buf;
+	void			*base;
+	uint64_t		offset;
+	size_t			size;
```

Comment:
Have you run the `send-recv` test?

Reply:
@nusislam no I forgot, I'll see to it again